Lib/uuid.py

```diff
@@ -656,7 +656,12 @@ def _random_getnode():
 
 _node = None
 
-def getnode():
+_node_getters_win32 = [_windll_getnode, _netbios_getnode, _ipconfig_getnode]
+
+_node_getters_unix = [_unix_getnode, _ifconfig_getnode, _ip_getnode,
+                      _arp_getnode, _lanscan_getnode, _netstat_getnode]
```

Comment:
Since these are constants, what do you think about capitalizing their names?  E.g. `_NODE_GETTERS_WIN32` and `_NODE_GETTERS_UNIX` (or maybe `_NODE_GETTERS_POSIX`)?